packages/jupyter-ai/jupyter_ai/tests/doc_loaders/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the collect_files and split functions in directory.py."""
```

Comment:
Note how other test files / test modules are named in `packages/jupyter-ai/jupyter_ai/tests/`. To be in line with this, remove this file, remove `doc_loaders` folder, rename `test_eligible_files.py` into `test_directory.py` and put it into `packages/jupyter-ai/jupyter_ai/tests/` folder.

Reply:
@andrii-i All comments addressed. Thank you for the excellent guidance on this PR.